test/models/network.js

```diff
@@ -223,6 +223,28 @@ describe("Network", function () {
 			ClientCertificate.remove(network.uuid);
 			Helper.config.public = true;
 		});
+
+		it("should remove client certs if there is a STS policy", function () {
```

Comment:
it **should remove** client certs?

>expect(ClientCertificate.get(network.uuid)).to.deep.equal(client_cert); // Should be unchanged

Well :wink: I think there's a not missing no?